src/BloomBrowserUI/bookEdit/readerSetup/readerSetup.ui.ts

```diff
@@ -758,7 +758,12 @@ function attachEventHandlers(): void {
       return false;
     });
 
-    $('#allowed-words-file-div').onOnce('mouseenter', function() { $(this).find('a').show(); });
+    $('#allowed-words-file-div').onOnce('mouseenter', function() {
+      var title = localizationManager.getText('ReaderSetup.RemoveWordList', 'Remove from this stage');
```

Comment:
Here I would use `var title = document.getElementById('remove_word_list').innerHTML;` so there won't be an extra call to the server.